server/models/availability.js

```diff
@@ -0,0 +1,62 @@
+const mongoose = require("mongoose");
+
+const daySchedule = new mongoose.Schema({
+  isAvailable: {
+    type: Boolean,
+    default: false,
+  },
+  startTime: {
+    type: Number,
+    min: 0,
+    max: 23,
```

Comment:
Max should be 24?

Reply:
Hey @moffatethan, I was considering 24 hour clock which I think we where using, which I believe goes from 00 to 23 in the night!

Reply:
How do we allow for minutes then? `23:59`

Reply:
We probably could've done a better implementation here since this implementation doesn't allow for minutes. But not a high priority right now.